man/avahi-daemon.conf.5.xml.in

```diff
@@ -201,10 +201,13 @@
       ("yes" or "no"). Enable wide-area DNS-SD, aka
       DNS-SD over unicast DNS. If this is enabled only domains
       ending in .local will be resolved on mDNS, all other domains
-      are resolved via unicast DNS. If you want to maintain multiple
-      different multicast DNS domains even with this option enabled
-      we encourage you to use subdomains of .local, such as
-      "kitchen.local". This option defaults to "no".</p>
+      are resolved via unicast DNS. When this is enabled, unless
+      explicitly specified reverse lookups will go over unicast DNS
+      and fall back to mDNS if unicast DNS lookups fails.
```

Comment:
Apparently I didn't change "fails" to "fail" when I replaced "lookup" with "lookups". My bad.

To avoid sending a PR with just one fix I'll take some changes from https://github.com/avahi/avahi/pull/481 that shouldn't prevent large PRs from being easily rebased and also add links to codespell, coverity, oss-fuzz reports to make it easier to find them.

Reply:
-> https://github.com/avahi/avahi/pull/562